app/components/forms/events/view/create-access-code.js

```diff
@@ -0,0 +1,93 @@
+import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
+
+const { Component, computed } = Ember;
+
+export default Component.extend(FormMixin, {
+  getValidationRules() {
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+
+      fields: {
+        accessCode: {
```

Comment:
The validation for `accessCode` is no more working as a result the form gets submitted even if the input is empty.
![screenshot from 2017-07-03 23-43-28](https://user-images.githubusercontent.com/12807846/27803884-7105f628-6049-11e7-9fd3-b72ffb50367d.png)